.github/workflows/tag-makedist.yaml

```diff
@@ -0,0 +1,72 @@
+name: make `make dist` tarball
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - release/**
+      - v**
+
+jobs:
+  amazonlinux:
+    strategy:
+      matrix:
```

Comment:
Why do these need to be a matrix?

Reply:
I don't like using environment variables in github actions because of quoting issues.